tests/support/loadJsdom.ts

```diff
@@ -31,6 +31,13 @@ global.requestAnimationFrame = (cb: (...args: any[]) => {}) => {
 
 global.cancelAnimationFrame = () => {};
 
+global.IntersectionObserver = function () {
```

Comment:
Have we thought about bringing the jsdom part to widget-core? Would be nice to not have the circular.